oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -690,3 +696,5 @@ Component-Server-Mappings:
   ClickHouse-jdbc-driver: ClickHouse
   Apache-Kylin-jdbc-driver: Apache-Kylin
   MysqlClient: Mysql
+  MicronautHttpClient: Micronaut
+  MicronautHttpServer: Micronaut
```

Comment:
```suggestion
  MicronautHttpClient: MicronautHttpServer
```

You should either set a mapping like this, according to your ID registration, but this would not match your UI side icon name: MICRONAUT.png.

The server side name should match the icon's file name.

Reply:
There is only one way to show the icon, the component name is the icon name.

Please update this file according, and post your screenshot after you set everything correctly.

Reply:
Sorry ,I should be stricter